src/main/java/com/inamik/text/tables/Cell.java

```diff
@@ -37,7 +37,7 @@
 
 public final class Cell
 {
-    public static final Collection<String> EMPTY = Collections.<String>emptyList();
+    public static final Collection<String> EMPTY = Collections.emptyList();
```

Comment:
Does this break Java 1.6 compatibility?

If not, then I would considering keeping your changes. but my recollection was that they would break in Java 1.6.

Reply:
They would break. After I revert to java 1.6 I will check if all the tests will run